Library/Homebrew/formula_installer.rb

```diff
@@ -877,6 +879,9 @@ def sanitized_argv_options
     args << "--debug" if debug?
     args << "--cc=#{@cc}" if @cc
     args << "--keep-tmp" if keep_tmp?
+    args << "--debug-symbols" if debug_symbols?
+    # Avoids dependecy error on flag
+    args << "--build-from-source" if build_from_source? && debug_symbols?
```

Comment:
This `--build-from-source` arg isn't used inside of here. But because `--debug-symbols` is dependent on it, an error occurs if it isn't set. This is less than glorious, I know.

Reply:
I see. Shouldn't you just pass both if `debug_symbols?` is `true`?

Reply:
Sounds good. Will make the change.

Reply:
Done in 91065b9ddd24bcc3c279b1b120ec34a2b3810f8b